### ISSUE TITLE
Make a single all-in-one Debian package

### ISSUE THREAD

**Issue by oss-assist** [Eloston/ungoogled-chromium#841]
A single Debian package would make it easier to download and install ungoogled Chromium; so that one package would hold the main stuff and another would contain the debugging symbols.

**Comment by Eloston** [comment]
I don't think ungoogled-chromium-debian should deviate too far from Debian's Chromium packaging. If we want to implement this, I think this should be implemented in ungoogled-chromium-portablelinux. That way, the resulting `.deb` package will be usable by any Debian-based distro. I believe there should be some tool that would make this easy to integrate with our existing build scripts?

**Comment by oss-assist** [comment]
@Eloston I don't see any Debian packaging config here? so how are the debs built?
fethcing the whole Chromium source?

**Comment by Eloston** [comment]
@oss-assist 

> I don't see any Debian packaging config here?

That's correct, there isn't any packaging config. I am proposing that we add packaging config to ungoogled-chromium-portablelinux.
